src/zellij.ts

```diff
@@ -0,0 +1,270 @@
+const sessions: Fig.Generator = {
+  script: "zellij list-sessions",
+  splitOn: "\n",
+};
+
+const completionSpec: Fig.Spec = {
+  name: "zellij",
+  description: "A terminal workspace with batteries included",
+  subcommands: [
+    {
+      name: "attach",
+      description: "Attach to a session",
+      subcommands: [
+        {
+          name: "options",
+          description: "Change the behavior of zellij",
+        },
+      ],
+      args: {
+        name: "session",
+        description: "The session to attach to",
+        generators: sessions,
+      },
+      options: [
+        {
+          name: ["-c", "--create"],
+          description: "Create a session if one does not exist",
+          args: {
+            name: "<session>",
+            description: "New session name",
+          },
+        },
+        {
+          name: "--index",
+          description:
+            "Number of the session index in the active sessions ordered by creation date",
+          args: {},
+        },
+      ],
+    },
+    {
+      name: "kill-all-sessions",
+      description: "Kill all sessions",
+      options: [
+        {
+          name: ["-y", "--yes"],
+          description: "Automation yes to prompts",
```

Comment:
I think having this would make sense here
```suggestion
          description: "Automation yes to prompts",
          isDangerous: true
```